lib/inspec/resources/yum.rb

```diff
@@ -59,7 +59,7 @@ def repositories
         # detect repo start
         in_repo = true if line =~ /^\s*Repo-id\s*:\s*(.*)\b/
         # detect repo end
-        if line == "\n" && in_repo
+        if (line == "\n" || line =~ /\s*Total packages:/) && in_repo
```

Comment:
I discovered this was still needed because of how the new output from CentOS 8 changes. If you revert this line you will see how it fails. Good thing I improved the tests to catch this :D

Reply:
It's likely not failing but adding extra data on the last repository that shouldn't be there. If you revert this line and run the test you'll see that it appends ``"Total packages" => "7,644"`` to the ``extras`` repository which shouldn't be there.